### PR TITLE
Add repoclosures to package_manifest

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
sorry @ehelms, I needed a guineapig PR :)

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

File: repoclosure/yum_f27.conf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# repos
+[f27-base]
+name=BaseOS
+enabled=1
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=fedora-27&arch=x86_64
+failovermethod=priority
+
+[f27-puppet-5]
+name=f27-puppet-5
+baseurl=http://yum.puppetlabs.com/puppet5/fedora/27/$basearch/
```
Comment (ekohl) [inline repoclosure/yum_f27.conf]: Should this be hardcoded to x86_64 too?

**Comment by ekohl** [comment]
It'd simplify jenkins repoclosure jobs if we did similar repoclosure changes to the existing branches so we can always refer to koji from the config file.

**Comment by ehelms** [comment]
@ekohl Dropped the basearch usage. By similar repoclosure changes, do you mean using obal for repoclosure everywhere?

**Comment by ekohl** [comment]
> By similar repoclosure changes, do you mean using obal for repoclosure everywhere?

I thought it could clean up the repoclosure job in jenkins if we added more repos in the release branches but it looks like that wouldn't help much because a lot of the code is unused.

**Pull request merged**